### PR TITLE
Indicate support for `doctrine/persistence` 3

### PR DESCRIPTION
**Pull request by derrabus**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
-        "doctrine/common": "^3.0.3",
+        "doctrine/common": "^3.0.3@dev",
         "doctrine/dbal": "^2.13.1 || ^3.2",
         "doctrine/deprecations": "^0.5.3",
         "doctrine/event-manager": "^1.1",
         "doctrine/inflector": "^1.4 || ^2.0",
         "doctrine/instantiator": "^1.3",
         "doctrine/lexer": "^1.2.3",
-        "doctrine/persistence": "^2.4",
+        "doctrine/persistence": "^2.4 || ^3@dev",
```
Comment (derrabus) [inline composer.json]: If you are fine with this PR, I'd suggest that we release Common 3.3 and Persistence 3.0 before merging it.

File: lib/Doctrine/ORM/Exception/NotSupported.php [hunk 2]
```diff
@@ -15,25 +15,23 @@ public static function create(): self
...
+        $message = <<<'EXCEPTION'
 Context: %s
 Problem: Feature was deprecated in doctrine/dbal 2.x and is not supported by installed doctrine/dbal:3.x
 Solution: See the doctrine/deprecations logs for new alternative approaches.
-EXCEPTION,
-            $context
-        ));
+EXCEPTION;
+
+        return new self(sprintf($message, $context));
```
Comment (derrabus) [inline lib/Doctrine/ORM/Exception/NotSupported.php]: Apparently, the way we used HEREDOC syntax in this file was incompatible with PHP 7.2.

Comment (greg0ire) [reply lib/Doctrine/ORM/Exception/NotSupported.php]: Yes, but you don't need to create a variable for this, having the comma on its own line should be enough, I'm going to amend this PR to fix that

**Review by greg0ire — DISMISSED** [review]

File: composer.json (hunk 1 above)
Comment (derrabus) [reply composer.json]: Done.

**Pull request merged**